src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -533,35 +533,36 @@ where
         let mut ticker = tokio::time::interval(timestamp_interval);
         ticker.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Skip);
 
-        let reclock_to_latest = dyncfgs::STORAGE_RECLOCK_TO_LATEST.get(&config.config.config_set());
+        let mut prev_binding_ts = mz_repr::Timestamp::MIN;
+        let mut prev_source_upper = Antichain::from_elem(FromTime::minimum());
 
-        let mut prev_probe: Option<Probe<FromTime>> = None;
         let timestamp_interval_ms: u64 = timestamp_interval
             .as_millis()
             .try_into()
             .expect("huge duration");
 
         while !cap_set.is_empty() {
+            // Check the reclocking strategy in every iteration, to make it possible to change it
+            // without restarting the source pipeline.
+            let reclock_to_latest =
+                dyncfgs::STORAGE_RECLOCK_TO_LATEST.get(&config.config.config_set());
+            tracing::info!("reclock_to_latest: {reclock_to_latest}");
```

Comment:
This will probably spam the logs with this message. Do we need to log this?

Reply:
No, I just forgot to remove this 😅